pkg/cmd/taskrun/describe.go

```diff
@@ -84,6 +84,10 @@ or
 					if err != nil {
 						return err
 					}
+					if len(trs) == 0 {
```

Comment:
The build test is failing due to your use of `fmt.Sprintf`. Just use the following:

```go
fmt.Fprintf(s.Out, "No TaskRuns present in namespace %s", opts.Params.Namespace())
```

Reply:
I don't know what I was thinking, I have made the changes. Will push in a while.